packages/arb-token-bridge-ui/src/components/common/Header.tsx

```diff
@@ -288,7 +288,7 @@ function MobileExternalLink({
 }: React.AnchorHTMLAttributes<HTMLAnchorElement>) {
   return (
     <ExternalLink
-      className="arb-hover flex w-full flex-row flex-nowrap items-center space-x-4 p-4 py-3 text-2xl font-medium text-white"
+      className="arb-hover flex w-full flex-row flex-nowrap items-center space-x-4 px-6 py-3 text-2xl font-medium text-white"
```

Comment:
This padding changes the alignment of the icons. All header icons were aligned in a straight line previously.

### Before
<img height="200" alt="image" src="https://user-images.githubusercontent.com/7558499/228501682-d2660497-169c-458b-a65f-17f160976e63.png">

### After
<img height="200" alt="image" src="https://user-images.githubusercontent.com/7558499/228501805-d4378977-8c46-4d41-9076-649241ca8e7e.png">